src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -796,300 +796,6 @@ void ValidateComponent(EnergyPlusData &state,
     }
 }
 
-void CalcPassiveExteriorBaffleGap(EnergyPlusData &state,
```

Comment:
The fact that these were here made it so that when I changed `DataSurfaces.hh`, 200 HVAC modules also recompiled. So I moved it somewhere else.